### PR TITLE
[11.0][add][purchase_stock_picking_return_invoicing_force_invoiced]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This module forces the status of the purchase order to remain in 'Invoiced'
even when you have returned items into stock, if you indicated in the purchase
order that you wanted to force the invoiced status.

It is a glue module between the module 'Purchase Force Invoiced' and
'Purchase Stock Picking Return Invoicing'.

This PR also adds a hook  method to the module 'purchase_stock_picking_return_invoicing'.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
OK, thanks

File: purchase_stock_picking_return_invoicing_force_invoiced/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Stock Picking Return Invoicing Force Invoiced",
+    "summary": "Glue module between purchase_force_invoiced "
+               "and purchase_stock_picking_return_invoicing",
+    "version": "11.0.1.0.0",
+    "category": "Purchases",
+    "website": "https://github.com/OCA/account-invoicing",
+    "author": "Eficent,"
```
Comment (MiquelRForgeFlow) [inline purchase_stock_picking_return_invoicing_force_invoiced/__manifest__.py]:
```suggestion
    "author": "Eficent,"
```
trailing whitespace

**Review by MiquelRForgeFlow — COMMENTED** [review]
You should add `purchase-workflow` in the `oca_dependencies.txt` file.

**Review by AdriaGForgeFlow — APPROVED** [review]
Please attend travis flake8 error and @mreficent comment. Otherwise, LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM :+1:

File: oca_dependencies.txt [hunk 2]
```diff
@@ -1 +1 @@
-
+purchase-workflow
```
Comment (AaronHForgeFlow) [inline oca_dependencies.txt:1]: how possibly purchase_stock_picking_return_invoicing was merged without adding this? :open_mouth:

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**